pom.xml

```diff
@@ -87,27 +88,33 @@
         <dependency>
             <groupId>org.jboss.arquillian.container</groupId>
             <artifactId>arquillian-container-test-spi</artifactId>
+            <version>${version.arquillian}</version>
```

Comment:
Are these really necessary? They are brought in by default from arquillian BOM via dependency management.
I believe I removed all of these in the past because they are not necessary and confusing.

Reply:
You are right, I will update the PR, weirdly IDE warned about the missing version for these dependencies.